### DISCUSSION TITLE
Face-to-face meeting at OSSF NYC

### DISCUSSION THREAD

**Discussion by opoupeney** [finos/technical-oversight-committee#47] [General]
Hi @finos/technical-steering-committee, who else will be at the OSFF conference in New York next week?

**Comment by caradelia** [comment]
I will be attending. Should we all carve out some time to connect @ColinEberhardt?

**Reply by ColinEberhardt** [reply]
Great idea - yes, I'll be there

**Comment by ColinEberhardt** [comment]
Looking at the agenda, we could meet at one of the coffee breaks. The morning sessions always run over, so an afternoon break may be the better option.

How about we aim to meet at the 3:45pm - 4:15pm slot?

**Comment by stephengoldbaum** [comment]
Works for me, though I'll have to leave 15m early.

**Comment by jonfreedman** [comment]
I am not in NY for the conference so hope you all enjoy it, will we hold a Dec meeting or just re-convene in Jan?

**Reply by ColinEberhardt** [reply]
Good question, our typical slot is one of the last weeks of the month. Might be worth skipping Dec. We'll discuss when we meet today.

**Comment by billc123-jam** [comment]
I am Olivier. Is there a spot people are congregating to tonight?

**Comment by billc123-jam** [comment]
Sounds great to me Colin
